me.timschneeberger.GalaxyBudsClient.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: me.timschneeberger.GalaxyBudsClient
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: GalaxyBudsClient
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+finish-args:
+  - --socket=x11 # https://github.com/AvaloniaUI/Avalonia/issues/1243 😔
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --allow=bluetooth
+  - --system-talk-name=org.bluez.*
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=com.canonical.Unity.LauncherEntry
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.*
```

Comment:
why do you need to own this?

Reply:
To get the tray icon to work. Since com.discordapp.Discord's manifest is already containing that, I assume it's also OK for this app.

Reply:
is your app electron based? Because you assumption is likely incorrect.